extensions/rich_text_components/Math/protractor.js

```diff
@@ -38,11 +41,69 @@ var escapedJsonToObj = function(json) {
+    '235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210' +
+    ' 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183' +
+    ' 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 ' +
+    '174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343' +
+    ' 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 11' +
+    '4 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"/></g><g transfor' +
+    'm="translate(1256,0)"><path stroke-width="10" d="M56 237T56 250T70 270H3' +
+    '69V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T7' +
+    '07 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 ' +
+    '250Z"/></g><g transform="translate(2261,0)"><path stroke-width="10" d="M' +
+    '21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247' +
+    ' 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T' +
+    '312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q43' +
+    '2 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q36' +
+    '3 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T8' +
+    '9 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -14' +
+    '8Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290' +
+    ' -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 15' +
+    '0 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 3' +
+    '90 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 2' +
+    '78 41 278H27Q21 284 21 287Z"/><g transform="translate(504,362)"><path st' +
+    'roke-width="10" transform="scale(0.707)" d="M109 429Q82 429 66 447T50 49' +
+    '1Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315' +
+    'T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H' +
+    '449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 13' +
+    '7Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 40' +
+    '1T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 55' +
+    '0Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"/></g' +
+    '></g></g></svg>')
+
+
+};
 
 var expectComponentDetailsToMatch = async function(elem, rawLatex) {
   // TODO(Jacob): Check that the actual latex being displayed is correct.
   var mathComponent = await elem.getAttribute('math_content-with-value');
   expect(escapedJsonToObj(mathComponent).raw_latex).toBe(rawLatex);
+  var mathSvgImage = elem.element(by.css(
+    '.protractor-test-math-svg'));
+  await waitFor.visibilityOf(
+    mathSvgImage,
+    'Math SVG takes too long to load.');
+  var src = await mathSvgImage.getAttribute('src');
+  await request(src, function(error, response, body) {
+    expect(body.replace(/(\r\n|\n|\r|\t)/gm, '')).toBe(SVGTAGS[rawLatex]);
```

Comment:
Can you please add a code comment to explain the `replace` here?

Reply:
actually this was to remove any white spaces, but that is actually not needed so removed it